nvflare/edge/assessors/buff_device_manager.py

```diff
@@ -56,6 +163,11 @@ def update_available_devices(self, devices: Dict, fl_ctx) -> None:
             f"assessor got reported {len(devices)} available devices from child. "
             f"total num available devices: {len(self.available_devices)}",
         )
+        # add new devices to device_client_map
+        for device_id, device in devices.items():
+            client_name = device.to_dict().get(PropKey.CLIENT_NAME)
```

Comment:
we can instead store a client_to_device_map,

and then have another set/list total devices if we want to keep track?